local.yml

```diff
@@ -17,7 +17,6 @@ services:
       - .:/app/uobtheatre-api:z
     env_file:
       - ./.envs/.local/.django
-      - .env
```

Comment:
Can we leave this in. It provides a good way to override the environment defined in the tracked files on a local deployment.

Reply:
If a `.env` file is not found an error is thrown. If we want to add local .envs they should be shoved in .envs/.local

Reply:
having an .env in the project root to override the "typical" environment variables is standard practice (and we need to to add, for example, square details and change mail backends, etc), it makes it easy and clear for people rather than having to go into some complicated folder structure. Either way, can you implement some way that isn't tracked or something, instead of just removing it and leaving it